src/Packagist/Api/Client.php

```diff
@@ -165,9 +165,7 @@ protected function request($url)
 
         return $this->httpClient
             ->get($url)
-            ->send()
-            ->getBody(true)
-        ;
```

Comment:
Sorry, I've been busy with work. Not really used github much, I guess I'm supposed to make the suggested changes and then update the pull request somehow? Can I make the change locally rebase and force push, will that automatically update the pull request?